common-rendering/src/components/header.tsx

```diff
@@ -0,0 +1,244 @@
+import { space } from "@guardian/src-foundations";
+import { from, until } from "@guardian/src-foundations/mq";
+import { Theme, Pillar } from "@guardian/types"; // TODO: update to /libs when possible
+import {
+	sport,
+	news,
+	opinion,
+	lifestyle,
+	culture,
+	text,
+} from "@guardian/src-foundations/palette";
+
+type HeaderProps = {
+	theme: Theme;
+	title: string;
+	credit: string;
+	standfirst: ReactElement;
+	byline?: ReactElement;
+};
+
+type Colours = {
+	backgroundTop: string;
+	backgroundBottom: string;
+	creditColour: string;
+	accentColour: string;
+};
+
+const getBackgroundTop = (backgroundTop: string): SerializedStyles => css`
+	display: flex;
+	background-color: ${backgroundTop};
+	width: 100%;
+	padding: ${space[1]}px ${space[2]}px ${space[5]}px;
+
+	${from.mobileLandscape} {
+		padding: ${space[1]}px ${space[5]}px ${space[5]}px;
+	}
+
+	${from.desktop} {
+		padding: ${space[1]}px 0 ${space[5]}px;
+	}
+`;
+
+const titleStyles: SerializedStyles = css`
+	display: block;
+	${headline.small()};
+	color: ${text.ctaPrimary};
+`;
+
+const getCreditStyles = (creditColour: string): SerializedStyles => css`
+	display: block;
+	${headline.xxxsmall()};
+	color: ${creditColour};
+	margin-bottom: ${space[1]}px;
+
+	${from.desktop} {
+		position: absolute;
+		left: ${space[5]}px;
+		max-width: 180px;
+	}
+`;
+
+const getBackgroundBottom = (backgroundBottom: string, accentColour: string): SerializedStyles => css`
+	display: flex;
+	padding: ${space[1]}px ${space[2]}px;
+	background-color: ${backgroundBottom};
+	width: 100%;
+	min-height: 100px;
+	p,
+	a,
+	li {
+		color: ${text.ctaPrimary};
+		${headline.xxxsmall()};
+		line-height: 135%;
+		margin-bottom: ${space[2]}px;
+	}
+
+	li::before {
+		content: "";
+		background-color: ${accentColour};
+		width: 0.75rem;
+		height: 0.75rem;
+		border-radius: 0.5rem;
+		display: inline-block;
+		vertical-align: middle;
+		margin-right: 0.5rem;
+	}
+
+	li {
+		padding-left: 1.25rem;
+		text-indent: -1.25rem;
+	}
+	a {
+		text-decoration: none;
+		padding-bottom: 0.005rem;
+		border-bottom: 1px solid ${accentColour};
+	}
+
+	${from.mobileLandscape} {
+		padding: ${space[1]}px ${space[5]}px;
+	}
+
+	${from.desktop} {
+		padding: ${space[1]}px 0;
+	}
+`;
+
+const bylineStyles: SerializedStyles = css`
+	margin: ${space[5]}px 0 ${space[3]}px;
+`;
+
+const leftColStyles: SerializedStyles = css`
+	display: inline-block;
+	height: 100%;
+	width: 220px;
+	padding: 0 ${space[5]}px;
+
+	${until.desktop} {
+		display: none;
+	}
+`;
+
+const containerStyles: SerializedStyles = css`
+	width: 100%;
+
+	${from.desktop} {
+		max-width: 700px;
+	}
+`;
+
+const updatedStyles: SerializedStyles = css`
+	color: ${text.ctaPrimary};
+	${textSans.xxsmall()};
+`;
+
+// The following pulsing dot implementation is a placeholder for the PulsingDot component, which
+// we will ideally lift up from dcr to common-rendering
+const allowsAnimation = "@media (prefers-reduced-motion: no-preference)";
```

Comment:
would be nice to export this from somewhere at some point

Reply:
Yes, definitely. In the note above, I mentioned that this code will be replaced with the PulsingDot component once we can share code between common-rendering and the other projects. :-)

Reply:
Yeah, I think for now putting everything you need in the same file is fine. There's going to be an element of fixing forward regardless.